feature-libs/order/components/order-details/order-detail-items/order-consigned-entries/order-consigned-entries.component.html

```diff
@@ -4,15 +4,27 @@
     [cxOutletContext]="{ item: consignment }"
   >
     <div class="cx-list-header col-12">
-      <div class="cx-list-status">
-        <span *ngIf="consignment">
-          {{
-            'orderDetails.deliveryStatus_' + consignment?.status | cxTranslate
-          }}
-        </span>
-      </div>
-      <div *ngIf="consignment?.statusDate" class="cx-list-date">
-        <div>{{ consignment?.statusDate | cxDate }}</div>
+      <div class="cx-consignment-details">
+        <div class="cx-list-status-container">
+          <div *ngIf="consignment" class="cx-list-status">
+            {{
+              'orderDetails.deliveryStatus_' + consignment?.status | cxTranslate
+            }}
+          </div>
+          <div *ngIf="consignment?.statusDate" class="cx-list-date">
+            <div>{{ consignment?.statusDate | cxDate }}</div>
+          </div>
+        </div>
+
+        <ng-container
```

Comment:
use the "outlet" to render the pickup items.

Reply:
Done